File: python/1012.py
num1,num2,num3=input().split()
num1,num2,num3=float(num1),float(num2),float(num3)
raio=num3
pi=3.14159
triangulo_retangulo=(num1*num3)/2
circulo=pi*raio**2
trapezio=((num1 + num2)*num3)/2
quadrado=num2**2
retangulo=num1*num2
print(f"TRIANGULO: {triangulo_retangulo:.3f}")
print(f"CIRCULO: {circulo:.3f}")
print(f"TRAPEZIO: {trapezio:.3f}")
print(f"QUADRADO: {quadrado:.3f}")
print(f"RETANGULO: {retangulo:.3f}")

File: cpp/1012.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){

    double a,b,c; cin >> a >> b >> c;
    
    double triangulo = (a*c)/2;
    double circulo = 3.14159 * c * c;
    double trapezio = (a+b)*c/2;
    double quadrado = b * b;
    double retangulo = a * b;

    cout << "TRIANGULO: " << fixed << setprecision(3) << triangulo << "\n";
    cout << "CIRCULO: " << fixed << setprecision(3) << circulo << "\n";
    cout << "TRAPEZIO: " << fixed << setprecision(3) << trapezio << "\n";
    cout << "QUADRADO: " << fixed << setprecision(3) << quadrado << "\n";
    cout << "RETANGULO: " << fixed << setprecision(3) << retangulo << "\n";

}